### PR TITLE
Update Opcodes

### PR DESCRIPTION
**Pull request by Quexington**
I've updated the opcodes and their descriptions to match the current list of opcodes in https://github.com/Chia-Network/chia-blockchain/blob/59de4ffe9f98de62d281695d1f519d65ef2e2ece/chia/types/condition_opcodes.py

### CODE REVIEW

File: docs/doc2.md [hunk 1]
```diff
@@ -77,17 +77,22 @@ The OpCodes are split into two categories: *"this spend is only valid if X"* and
...
+* **CREATE_COIN_ANNOUNCEMENT - [60] - (60 message)**: If this spend is valid, this creates an ephemeral announcement with an ID dependent on the coin that creates it. Other coins can then assert an announcement exists for inter-coin communication inside a block.
+* **ASSERT_COIN_ANNOUNCEMENT - [61] - (61 0xannouncementID)**: This spend is only valid if there was an announcement in this block matching the announcementID.  The announcementID is the message that was announced concatenated with the coin ID of the coin that announced it `announcementID == (message + coinID)`.
+* **CREATE_PUZZLE_ANNOUNCEMENT - [62] - (62 message)**: If this spend is valid, this creates an ephemeral announcement with an ID dependent on the puzzle that creates it. Other coins can then assert an announcement exists for inter-coin communication inside a block.
+* **ASSERT_PUZZLE_ANNOUNCEMENT - [63] - (63 0xannouncementID)**: This spend is only valid if there was an announcement in this block matching the announcementID.  The announcementID is the message that was announced concatenated with the puzzle hash of the coin that announced it `announcementID == (message + puzzle_hash)`.
```
Comment (matt-o-how) [inline docs/doc2.md:87]: Should clarify that its a hash of those things concatenated

File: docs/doc2.md [hunk 2]
```diff
@@ -77,17 +77,22 @@ The OpCodes are split into two categories: *"this spend is only valid if X"* and
...
+* **AGG_SIG_ME - [50] - (50 0xpubkey 0xmessage)**: This spend is only valid if the attached aggregated signature contains a signature from the specified public key of that message concatenated with the coin's ID.
+* **CREATE_COIN - [51] - (51 0xpuzzlehash amount)**: If this spend is valid, then create a new coin with the given puzzlehash and amount.
+* **ASSERT_FEE - [52] - (52 amount)**: This spend is only valid if there is unused value in this transaction equal to *amount*, which is explicitly to be used as the fee.
+* **CREATE_COIN_ANNOUNCEMENT - [60] - (60 message)**: If this spend is valid, this creates an ephemeral announcement with an ID dependent on the coin that creates it. Other coins can then assert an announcement exists for inter-coin communication inside a block.
+* **ASSERT_COIN_ANNOUNCEMENT - [61] - (61 0xannouncementID)**: This spend is only valid if there was an announcement in this block matching the announcementID.  The announcementID is the message that was announced concatenated with the coin ID of the coin that announced it `announcementID == (message + coinID)`.
```
Comment (matt-o-how) [inline docs/doc2.md:85]: Should clarify that the ID is the hash of those things concatenated.

And also, its coinID + message

File: docs/doc2.md (hunk 1 above)
Comment (matt-o-how) [reply docs/doc2.md:87]: the order is `puzzle_hash + message`

Comment (Quexington) [reply docs/doc2.md:87]: Oh yes, do you think it should be Chialisp like `(sha256 coinID message)` or just some pseudo code like `hash(coinID + message)`

**Review by hoffmang9 — APPROVED** [review]

**Pull request merged**